.github/workflows/upload-models.yaml

```
@@ -1264,6 +1264,32 @@ jobs:
+        run: |
+          git lfs install
+          git clone https://huggingface.co/hojreh/Quds-v4-sherpa-onnx
+          cd Quds-v4-sherpa-onnx
+          rm -rf .git
+          rm -rf *.int8.onnx
+          rm -rf .gitattributes
+          tar cjfv sherpa-onnx-Quds-v4-fa.tar.bz2 *
+          ls -lh sherpa-onnx-Quds-v4-fa.tar.bz2
+
+      - name: Quds v4 Persian (INT8)
+        if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)